linuxNew/jdk/redhat/src/main/packaging/entrypoint.sh

```diff
@@ -5,6 +5,8 @@ set -euox pipefail
 # into it without changing its ownership (which could render the folder or its contents inaccessible to the host user),
 # add the user builder to the group with the same GID as the host user.
 HOST_USER_GID=$(stat -c "%g" /home/builder/out)
+# gpg: signing failed: Inappropriate ioctl for device
```

Comment:
Is this intended to indicate that this error occurs if the next line is not in place? Would be good to make that clear in the comment.

Reply:
updated